### PR TITLE
App authentication functions

### PR DESCRIPTION
**Pull request by NoahNsimbe**
# Feature
**_WHAT DOES THIS PR DO?_**
- [x] Functionality to check if a user with a given phone number exists

**_WHAT JIRA STORY/TASK IS RELATED TO THIS PR?_**
None

**_WHAT IS THE LINK TO THE PR BRANCH_**
[ft-app-auth-triggers](https://github.com/airqo-platform/AirQo-api/tree/ft-app-auth-triggers
)

**_HOW DO I TEST OUT THIS PR?_**
[Documentation](https://app.gitbook.com/o/-MCogOVGQqjUharJKchL/s/-MKqyQkcfs54GYi-q96G/firebase#check-if-user-exists)

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
None

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
None
 
**_ARE THERE ANY RELATED PRs?_**
None

### CODE REVIEW

File: src/firebase/auth/functions/index.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      "emailAddress": emailAddress,
+    };
+
+    // eslint-disable-next-line max-len
+    axios.post(endPoint, JSON.stringify(body), {headers: {"Content-Type": "application/json"}})
+        .then((res) => {
+          console.log(`Welcome message status code: ${res.status}`);
+        })
+        .catch((error) => {
+          console.error(error);
```
Comment (123MwanjeMike) [inline src/firebase/auth/functions/index.js:46]: We could also use npm's morgan to log these especially for production.

Comment (NoahNsimbe) [reply src/firebase/auth/functions/index.js:46]: Thanks @123MwanjeMike, shall add the appropriate logging functionality

**Review by Codebmk — APPROVED** [review]
@NoahNsimbe as per documentation, functionality seems solid

**Comment by NoahNsimbe** [comment]
Merging this. Functionality was deployed to  production 3 months ago. cc @123MwanjeMike @Baalmart

**Pull request merged**